<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="serialTab">
     <attribute name="title">
      <string>串口调试</string>
     </attribute>
     <widget class="QGroupBox" name="serialGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>串口配置</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>158</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="serialNumListLabel">
          <property name="text">
           <string>端口号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="serialNumListBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>波特率</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="baudRateBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dataBitsLabel">
          <property name="text">
           <string>数据位</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dataBitsBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="stopBitsLabel">
          <property name="text">
           <string>停止位</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitsBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="parityLabel">
          <property name="text">
           <string>校  验</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="parityBox"/>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="openButton">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="flowControlLabel">
          <property name="text">
           <string>流控制</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="flowControlBox"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="recvGroupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>611</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>接收区</string>
      </property>
      <widget class="QPlainTextEdit" name="recvPlainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>601</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="helpGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>161</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>辅助</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>160</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="hexSendcheckBox">
          <property name="text">
           <string>Hex发送</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="hexRecvcheckBox">
          <property name="text">
           <string>Hex显示</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="clearSendButton">
          <property name="text">
           <string>清发送区</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="clearRecvButton">
          <property name="text">
           <string>清接收区</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="saveFilecheckBox">
          <property name="text">
           <string>保存文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="sendGroupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>611</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>发送区</string>
      </property>
      <widget class="QTextEdit" name="sendTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>561</width>
         <height>61</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>20</y>
         <width>31</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="otherTab">
     <attribute name="title">
      <string>其它Tab</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
